### DISCUSSION TITLE
Path appears to be invalid: /storage/emulated/0/Download/myDocument.pdf

### DISCUSSION THREAD

**Discussion by jacksonazevedo** [afreakyelf/Pdf-Viewer#76] [General]
When clicking the download button I get the following message in logcat:

Path appears to be invalid: /storage/emulated/0/Download/myDocument.pdf

**Comment by afreakyelf** [comment]
Hi, Thank you for using this library. Above issue has been fixed in latest version.
https://github.com/afreakyelf/Pdf-Viewer/releases/tag/v2.0.1
